met-api/src/met_api/models/engagement.py

```diff
@@ -39,13 +40,34 @@ def get_engagement(cls, engagement_id):
         data = db.session.query(Engagement).filter_by(id=engagement_id).first()
         return engagement_schema.dump(data)
 
+    @classmethod
+    def get_published_engagement(cls, engagement_id):
+        """Get an engagement."""
+        engagement_schema = EngagementSchema()
+        data = db.session.query(Engagement)\
+            .filter(Engagement.status_id != Status.Draft.value)\
+            .filter_by(id=engagement_id)\
```

Comment:
you can simplify thiis with an and ;no need to of separate filter_by

Reply:
updated